.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
+
+root = true
+
+[*]
+charset = utf-8
```

Comment:
@witmicko I think we might just need to scope the `indent_size` to js files if we need a different indentation for some other kind of file like markdown or something.

Ironically this file itself doesn't have a final newline right now :joy: